File: pinyougou-sellergoods-interface/src/main/java/com/pinyougou/sellergoods/service/BrandService.java
package com.pinyougou.sellergoods.service;

import com.pinyougou.entity.PageResult;
import com.pinyougou.entity.Result;
import com.pinyougou.pojo.TbBrand;

import java.util.List;
import java.util.Map;

public interface BrandService {

    //查询所有
    public List<TbBrand> findAll();
    //分页查询
    PageResult findPage(Integer pageNum,Integer pageSize);
    //新增
    public void add(TbBrand tbBrand);


    //修改
    public void update(TbBrand tbBrand);
    //查一个
    public TbBrand findOne(Long id);




    //删除
    public void delete(Long [] ids);

    //分页条件查询
    PageResult findPage(TbBrand tbBrand, Integer pageNum,Integer pageSize);

    //下拉框
    public List<Map> selectOptionList();

}

File: pinyougou-manager-web/src/main/webapp/js/service/brandService.js
var sellergoods_url = "http://localhost:9101";
app.service("brandService", function ($http) {
    //品牌服务
    this.findAll = function () {
        return $http.get(sellergoods_url + "/brand/findAll.do")
    };

    this.findOne = function (id) {
        return $http.get(sellergoods_url + "/brand/findOne.do?id=" + id)
    };
    this.findPage = function (page, size) {
        return $http.get(sellergoods_url + "/brand/findPage.do?page=" + page + '&size=' + size)
    };

    this.dele = function (ids) {
        return $http.get(sellergoods_url + '/brand/delete.do?ids=' + ids)
    };

    this.search = function (page, size, searchEntity) {
        return $http.post(sellergoods_url + "/brand/search.do?page=" + page + '&size=' + size, searchEntity)
    };

    this.add = function (entity) {
        return $http.post(sellergoods_url + "/brand/add.do", entity)
    };

    this.update = function (entity) {
        return $http.post(sellergoods_url + "/brand/update.do", entity)
    }

    //下拉框
    this.selectOptionList = function () {
        return $http.get(sellergoods_url + "/brand/selectOptionList.do");
    }
});